### PR TITLE
Simple lint warning fixes

### PR DESCRIPTION
**Pull request by jimmynotjim**
Got tired of all the warnings, figured it'd be good to get down to the real ones (like complexity) so we know where to concentrate when hopefully refactoring some of the JS next month.

### CODE REVIEW

File: src/static/js/modules/init-chosen.js [hunk 1]
```diff
@@ -10,7 +10,9 @@ require( 'chosen' );
 function init() {
   $( '.chosen-select' ).chosen( {
     width:           '100%',
+    /* eslint-disable */
```
Comment (anselmbradford) [inline src/static/js/modules/init-chosen.js]: Ideally, say in the code comment what rule is being ignored.

File: src/static/js/modules/jquery/cf_notifier.js [hunk 2]
```diff
@@ -7,8 +7,11 @@
 var $ = require( 'jquery' );
 var handlebars = require( 'handlebars' );
 
-var _notifierTemplate = '<div class="cf-notification cf-notification__{{ state }}" style="display: none;">' +
-  '<span class="cf-notification_icon cf-notification_icon__{{ state }} cf-icon cf-icon-{{ icon }}-round"></span>' +
+var _notifierTemplate = '<div class="cf-notification cf-notification__{{ state }}" ' +
+                             'style="display: none;">' +
+  '<span class="cf-notification_icon ' +
```
Comment (anselmbradford) [inline src/static/js/modules/jquery/cf_notifier.js]: Is the indentation how you want it on L:11 and L:12?

File: src/static/js/modules/init-chosen.js (hunk 1 above)
Comment (jimmynotjim) [inline src/static/js/modules/init-chosen.js]: Ha, so I did that, but then it triggered three new warnings (including line length), so I gave up and went global.

File: src/static/js/modules/simple-table-row-links.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ function init() {
 function _tableClicked( event ) {
   var target = event.target;
   var tagName;
-  while ( target.tagName !== 'TR') {
+  while ( target.tagName !== 'TR' ) {
```
Comment (anselmbradford) [inline src/static/js/modules/simple-table-row-links.js]: Oops, my bad!

File: src/static/js/modules/jquery/cf_notifier.js (hunk 2 above)
Comment (jimmynotjim) [inline src/static/js/modules/jquery/cf_notifier.js]: I kinda matched how we indent our markup. Wasn't sure how to do this one really, so open to suggested examples.

File: src/static/js/modules/init-chosen.js (hunk 1 above)
Comment (anselmbradford) [inline src/static/js/modules/init-chosen.js]: So not awesome, but you can do either of the following which may provide some indication of why the ignore is there:

```
function init() {
  $( '.chosen-select' ).chosen( {
    width:           '100%',
    no_results_text: 'Oops, nothing found!' // eslint-disable-line camelcase, no-inline-comments, max-len
  } );
}
```

…or…

```
function init() {
  $( '.chosen-select' ).chosen( {
    width:           '100%',
    /* eslint-disable camelcase, lines-around-comment */
    no_results_text: 'Oops, nothing found!'
    /* eslint-endable */
  } );
}
```

Comment (anselmbradford) [inline src/static/js/modules/init-chosen.js]: Not awesome in that you have to ignore the additional issues that come up, but at least the first rule being ignored is the real one.

File: src/static/js/modules/jquery/cf_notifier.js (hunk 2 above)
Comment (anselmbradford) [inline src/static/js/modules/jquery/cf_notifier.js]: Hm, okay, it's fine for now I think, but we may want to find ways to have the templates in separate .tmpl files in the future.

**Comment by anselmbradford** [comment]
:+1: Just the eslint ignore line as you see fit (I prefer single line ignores myself because there's less to remove if/when they're refactored away).

**Comment by anselmbradford** [comment]
I'd say https://github.com/cfpb/cfgov-refresh/issues/300 can be closed after this. Linting warnings will be an ongoing thing.

File: src/static/js/modules/jquery/cf_notifier.js (hunk 2 above)
Comment (jimmynotjim) [inline src/static/js/modules/jquery/cf_notifier.js]: Yep, that is the plan going forward as I take this out of FJ and into CF (I'm about mid-way done but haven't had time to finish).

File: src/static/js/modules/init-chosen.js (hunk 1 above)
Comment (jimmynotjim) [inline src/static/js/modules/init-chosen.js]: Went with your second one so I'm only adding one additional warning to suppress 4ebb5d0

**Comment by sebworks** [comment]
:+1:

**Pull request merged**